jsapp/js/account/usage/usageProjectBreakdown.module.scss

```diff
@@ -1,7 +1,96 @@
 @use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
 
 .root {
   width: auto;
-  padding-top: sizes.$x40;
+  padding-bottom: sizes.$x40;
+  padding-top: sizes.$x20;
   margin: 0 sizes.$x40;
+  height: 100%;
+  overflow-x: auto;
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.projects {
+  padding-right: 15%;
+  padding-left: 0%;
```

Comment:
This might need to get changed to `sizes.$x8` to stay flush with the rest of the column.